src/polyswarm_api/types/resources.py

```diff
@@ -0,0 +1,619 @@
+import logging
+import os
+import tempfile
+from binascii import unhexlify
+from enum import Enum
+from hashlib import sha256 as _sha256, sha1 as _sha1, md5 as _md5
+
+from future.utils import raise_from, string_types
+from ordered_set import OrderedSet
+
+from polyswarm_api.const import FILE_CHUNK_SIZE
+
+try:
+    import yara
+except ImportError:
+    yara = None
+
+from polyswarm_api import exceptions
+from polyswarm_api import const
+from . import base
+from . import schemas
+from . import date
+
+logger = logging.getLogger(__name__)
+
+
+#####################################################################
+# Resources returned by the API
+#####################################################################
+
+class PolyScore(base.BasePSJSONType, base.BasePSResourceType):
+    SCHEMA = schemas.polyscore_schema
+
+    def __init__(self, json, polyswarm=None):
+        super(PolyScore, self).__init__(json=json, polyswarm=polyswarm)
+
+        self.scores = json['scores']
+
+    def get_score_by_id(self, instance_id):
```

Comment:
It used to return the scores for all the instances in a `Submission`. Now that we got rid of the `Submission` abstraction, I think we can change this. Also the `polyscore` is being calculated in the instance serialization on the server-side, so it is unlikely we would be needing this in the future. Should I go ahead and assume this will be already available in the `ArtifactInstance` and remove this?

Reply:
I think this is the correct move. Let's make sure it's in the instance, provided calculating it is cheap enough, which I believe it is.

Reply:
Sounds good. Will work on it :+1: